.changeset/silly-suns-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/util': minor
+---
+
+allow users to specify their environment as Node or Browser to override Firebase's runtime environment detection and force the SDK to act as if it were in the respective environment
```

Comment:
Since we're referring to actual string literals `node` and `browser`, I would suggest representing them as they will be used, with lowercase and backticks. Maybe also punctuation (capital and period), I know nobody really reads our CHANGELOG.md files but they're technically public.